libnavigation-base/src/test/java/com/mapbox/navigation/base/options/NavigationOptionsTest.kt

```diff
@@ -70,6 +70,12 @@ class NavigationOptionsTest : BuilderTest<NavigationOptions, NavigationOptions.B
             .routeRefreshOptions(mockk())
             .routeAlternativesOptions(mockk())
             .incidentsOptions(mockk())
+            .historyRecorderOptions(
+                HistoryRecorderOptions.Builder()
+                    .enabled(false)
+                    .fileDirectory("123")
+                    .build()
```

Comment:
```suggestion
                mockk()
```

Reply:
some reason this wasn't working 🤔

Reply:
this was still something i wanted to figure out because other objects seem to be fine.

but this class may have an exception because the default types are the default primitive values. where other objects here, have defaults that are not primitive defaults

``` kotlin
        private var enabled: Boolean = false
        private var fileDirectory: String? = null
```
is the same as
``` java
        private boolean enabled;
        private String fileDirectory
```

Reply:
What happens if you set explicitly the defaults? Does it work?

Reply:
the test requires different values, so the test fails when you set a default value. mockk() probably initializes by defaults, meaning, all the other mockk() functions are working by luck